recipe/meta.yaml

```
@@ -1,26 +1,27 @@
 {% set name = "pcre2" %}
-{% set version = "10.42" %}
+{% set version = "10.46" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://github.com/PCRE2Project/{{ name }}/archive/refs/tags/{{ name }}-{{ version }}.tar.gz
-  sha256: a8e52a9bd1bca8f51c5c24823adc2a99acb12288e289a6507090c1a4a4815010
+  sha256: ffde0c6ead457c4f8edcd8f9ce490bbabd30adabdfd179ecaddd320946cf3872
 
 build:
-  number: 1
+  number: 0
   run_exports:
     # https://abi-laboratory.pro/index.php?view=timeline&l=pcre2
-    - {{ pin_subpackage("pcre2", max_pin="x.x") }}
+    - {{ pin_subpackage(name, max_pin="x.x") }}
 
 requirements:
   build:
+    - {{ stdlib('c') }}
     - {{ compiler('c') }}
     - cmake
-    - ninja
-    - posix  # [win]
+    - ninja-base
+    - posix   # [win]
     - m2-sed  # [win]
```

```suggestion
    - msys2-sed  # [win]
```
The `msys2-*` packages are the newer and more preferred ones to use on Windows.

---

When using it, Windows build throws an error:

```
Could not solve for environment specs
The following packages are incompatible
├─ msys2-sed =* * is installable and it requires
│  └─ m2-sed <0a0 *, which can be installed;
└─ posix =* * is not installable because it requires
   └─ m2-base =* *, which requires
      └─ m2-sed =* *, which conflicts with any installable versions previously reported.
```

---

Looks like `msys2-posix` is the equivalent in the new preferred packages.

---

`msys2-sed` worked alongside with `msys2-posix`, thanks!